### PR TITLE
KSPM Getting-started requirements update 

### PR DESCRIPTION
**Pull request by tinnytintin10**
Users won't be able to view posture data if they don't have read permissions over posture indices. This PR adds this requirement in the requirements section of the getting started section of our docs. 

We've also made updates to our UI here https://github.com/elastic/kibana/pull/145794 that will guide users towards updating their permissions.

### CODE REVIEW

**Comment by tinnytintin10** [comment]
@benironside feel free to update the wording or formatting of the message. Also, I have asked for your review on the type we used for the UI updates in the PR I linked above.

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 1]
```diff
@@ -6,6 +6,7 @@ This page explains how to configure the Kubernetes Security Posture Management (
 [sidebar]
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, ensure you have the `read` privilege to the following elasticsearch indices: \n- `logs-cloud_security_posture.findings_latest-*`\n- `logs-cloud_security_posture.scores-*`
```
Comment (tehilashn) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:9]: I think it's also worth adding logs-cloud_security_posture.findings so they can view that index in discover (and for future history features, debugging and so on).

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 2]
```diff
@@ -6,6 +6,7 @@ This page explains how to configure the Kubernetes Security Posture Management (
 [sidebar]
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, ensure you have the `read` privilege to the following elasticsearch indices: \n- `logs-cloud_security_posture.findings_latest-*`\n- `logs-cloud_security_posture.scores-*` \n- `logs-cloud_security_posture.findings`
```
Comment (benironside) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:9]:
```suggestion
* To view posture data, ensure you have the `read` privilege for the following {es} indices:

- `logs-cloud_security_posture.findings_latest-*`
- `logs-cloud_security_posture.scores-*` 
- `logs-cloud_security_posture.findings`
```

**Review by benironside — COMMENTED** [review]
I'm making some minor formatting changes

**Pull request merged**